--- mmm-0.1.0/.mm/bin.orig	2016-12-19 19:19:18.000000000 +0100
+++ mmm-0.1.0/.mm/bin	2018-03-24 15:06:16.813108957 +0100
@@ -12,11 +12,11 @@ BIN_CFLAGS += $(shell test "$(BIN_PKGMOD
 BIN_LD_FLAGS += $(shell test "$(BIN_PKGMODULES)" 2>/dev/null && pkg-config $(BIN_PKGMODULES) --libs)
 
 $(BINARY): $(BIN_CFILES) $(LIBNAME_A)
-	@echo "CCLD" $@; $(CC) $(SYSROOT) -Ilib -I .. $(BIN_CFLAGS) $(BIN_LD_FLAGS)  \
+	echo "CCLD" $@; $(CC) $(SYSROOT) -Ilib -I .. $(BIN_CFLAGS) $(BIN_LD_FLAGS)  \
 		$(BIN_CFILES) $(LIBNAME_A) -o $@
 
 $(BINARY).efence: $(BIN_CFILES) $(LIBNAME_A) 
-	@echo "CCLD" $@; $(CC) $(SYSROOT) -Ilib -I .. $(BIN_CFLAGS) $(BIN_LD_FLAGS) \
+	echo "CCLD" $@; $(CC) $(SYSROOT) -Ilib -I .. $(BIN_CFLAGS) $(BIN_LD_FLAGS) \
 		$(BIN_CFILES) $(LIBNAME_A) -lefence -o $@
 
 all: $(BIN_TARGETS)
--- mmm-0.1.0/.mm/lib.orig	2018-03-24 15:06:16.813108957 +0100
+++ mmm-0.1.0/.mm/lib	2018-03-24 15:08:07.469799580 +0100
@@ -16,23 +16,23 @@ LIB_LD_FLAGS += $(shell test "$(LIB_PKGM
 LIBNAME_A=$(LIBNAME).a
 
 $(LIBNAME).a: $(LIB_OBJS)
-	@echo "  AR" $@;ar rcs $@ $(LIB_OBJS)
+	echo "  AR" $@;ar rcs $@ $(LIB_OBJS)
 
 CLEAN_FILES += symbols
 
 symbols: Makefile $(LIB_OBJS)
-	@echo " gen" $@;  nm -a $(LIB_OBJS) | grep " T " | cut -f 3 -d ' ' | grep "^$(SYMBOL_PREFIX)" | grep -v "^_" | sort > $@
+	echo " gen" $@;  nm -a $(LIB_OBJS) | grep " T " | cut -f 3 -d ' ' | grep "^$(SYMBOL_PREFIX)" | grep -v "^_" | sort > $@
 
 $(LIBNAME).$(LIBEXT): $(LIB_OBJS) symbols
-	@echo "  LD" $@;$(CC) --shared $(LIB_OBJS) -o $@ $(LIB_LD_FLAGS)
+	echo "  LD" $@;$(CC) --shared $(LIB_OBJS) -o $@ $(LIB_LD_FLAGS)
 
 # this rule is generic and could also belong with bin..
 # we try to be overly generic and throw in the cflags of both lib and bin..
 %.o: %.c *.h $(LIB_HFILES)
-	@echo "  CC" $@;$(CC) $(SYSROOT) -c $(LIB_CFLAGS) $< -o $@
+	echo "  CC" $@;$(CC) $(SYSROOT) -c $(LIB_CFLAGS) $< -o $@
 
 lib/%.o: lib/%.c lib/*.h $(LIB_HFILES)
-	@echo "  CC" $@;$(CC) $(SYSROOT) -c $(LIB_CFLAGS) $< -o $@
+	echo "  CC" $@;$(CC) $(SYSROOT) -c $(LIB_CFLAGS) $< -o $@
 
 LIBDIR ?= $(PREFIX)/lib
 #$(P64)
--- mmm-0.1.0/.mm/pkgconfig.orig	2016-12-19 19:19:18.000000000 +0100
+++ mmm-0.1.0/.mm/pkgconfig	2018-03-24 15:06:16.813108957 +0100
@@ -6,7 +6,7 @@ $(PROJECT_NAME).pc:
 	@echo " gen" $@
 	@echo "prefix=$(PREFIX)"     > $@
 	@echo exec_prefix=\$${prefix} >> $@
-	@echo libdir=\$${prefix}/lib >> $@
+	@echo "libdir=$(LIBDIR)" >> $@
 	@echo includedir=\$${prefix}/include >> $@
 	@echo apiversion=$(API_VERSION) >> $@
 	@echo >> $@
